cli/app.go

```diff
@@ -425,32 +425,114 @@ var app = &cli.App{
 					Subcommands: []*cli.Command{
 						{
 							Name:  "add",
```

Comment:
[nit] How do you feel about naming this `add-from`? More verbose, but just thinking that `dataset add filter` would expect the filter right after, whereas you also need to provide the dataset ID.

Reply:
I feel like it's reasonable to have to provide the dataset ID as part of the dataset command?